pkg/server/clock_monotonicity.go

```diff
@@ -187,16 +186,19 @@ func periodicallyPersistHLCUpperBound(
 
 	for {
 		select {
-		case persistInterval, ok = <-persistHLCUpperBoundIntervalCh:
```

Comment:
NB: this channel is never closed (we have a dedicated stop channel), so I removed the `ok` variable.